src/popup/router/components/InputAmount.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <InputField
-    class="amount-input"
+    class="input-amount"
```

Comment:
Why did we do it this way around? It sounds wrong to me tbh, usually we put the description first and then the type. Aka `mainButton`, `navLink` (...) so I would expect this to be `AmountInput`.

Reply:
Throughout the project we are differentiating components by their purpose: this way they are near each other if files are sorted lexicographically. For example: `NameItem, NameRow`, `AuctionCard, AuctionOverview`, for pages examples are: `AuctionBid, AuctionList, AuctionHistory`, `PermissionsSettings, PermissionsDetails` and many others

Reply:
So now I have `InputAmount` and `LabelAmount` and `ButtonAmount`?

Reply:
> So now I have `InputAmount` and `LabelAmount` and `ButtonAmount`?

Yes. Actually we are following this guide section https://vuejs.org/v2/style-guide/#Tightly-coupled-component-names-strongly-recommended

Reply:
But then they are not coupled at all? If i do this I have to search them in Input in Label and Button? I think the guide you quoted suggests the other way around AmountInput AmountButton AmountLabel so that these are close together.

Reply:
I agree with @etharner, maybe he've sent not the right link. 
I would say this rule is explaining this situation
https://vuejs.org/v2/style-guide/#Order-of-words-in-component-names-strongly-recommended
And according to this rule we called the components everywhere, exept for several places, some of them @etharner have fixed (maybe everyone).
Also we were naming components this way in the base-aepp aswell.

Reply:
I mean you are the maintainers, if you both agree on this I am not going to stand in your way :)